iothub/device/tests/Pipeline/RetryDelegatingHandlerTests.cs

```diff
@@ -221,14 +221,18 @@ public async Task DeviceNotFoundExceptionReturnsDeviceDisabledStatus()
             var sut = new RetryDelegatingHandler(contextMock, nextHandlerMock);
 
             // act
-            await ((Func<Task>)(() => sut
-                .OpenAsync(CancellationToken.None)))
-                .ExpectedAsync<IotHubClientException>()
-                .ConfigureAwait(false);
+            Func<Task> act = async () =>
+            {
+                await sut
+                .OpenAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
Line length not too long to require breaking
```suggestion
                await sut.OpenAsync(CancellationToken.None).ConfigureAwait(false);
```

Reply:
Yeah, I missed it